cats/optimise_starttime.py

```diff
@@ -0,0 +1,34 @@
+"""
+Timeseries conversion
+"""
```

Comment:
Is "timeseris conversion" making sense now?

Reply:
Removed in ce0c813c03d82bd00fcf8ca35cd791a4953dee86

Reply:
Removed in ce0c813c03d82bd00fcf8ca35cd791a4953dee86